client/src/app/+about/about-peertube/about-peertube.component.html

```diff
@@ -0,0 +1,94 @@
+<div i18n class="about-peertube-title">
+  About PeerTube
+</div>
+
+<div class="description">
+  <p i18n>PeerTube is a federated (ActivityPub) video streaming platform using P2P (WebTorrent) directly in the web browser.</p>
+
+  <p i18n>
+    It is a free and open-source software, under the <a href="https://github.com/Chocobozzz/PeerTube/blob/develop/LICENSE">AGPLv3 licence</a>.
+  </p>
+
+  <p i18n>
+    For more information, please visit <a target="_blank" rel="noopener noreferrer" href="https://joinpeertube.org">joinpeertube.org</a>.
+  </p>
+</div>
+
+<div id="p2p-privacy">
+  <div i18n class="section-title">P2P & Privacy</div>
+
+  <p i18n>
+    PeerTube uses the BitTorrent protocol to share bandwidth between users.
+    This implies that your IP address is stored in the instance's BitTorrent tracker as long as you download or watch the video.
+  </p>
+
+  <h4 i18n class="p2p-privacy-title">What are the consequences?</h4>
+
+  <p i18n>
+    In theory, we could imagine that someone with enough technical skills may create a script that tracks which IP is downloading which video.
```

Comment:
"In theory, someone with enough technical skills could create a script that tracks which IP is downloading which video."